src/elm/Page/Community/Objectives.elm

```diff
@@ -85,6 +85,13 @@ view ({ shared } as loggedIn) model =
                     Page.fullPageGraphQLError (t "community.objectives.title_plural") e
 
                 Loaded community ->
+                    let
+                        isAllActionsCompleted =
+                            community.objectives
+                                |> List.map .actions
+                                |> List.map (\ac -> List.all (\a -> a.isCompleted) ac)
+                                |> List.head
```

Comment:
this is actually a good way of measuring if an objective is completed but the backend already does that for you. Take a look at the objective type:

<img width="294" alt="Captura de Tela 2021-03-01 às 14 01 18" src="https://user-images.githubusercontent.com/1082127/109531407-9b0dfb80-7a96-11eb-81dd-0b3434e2740c.png">

Her you can se the `isCompleted` property that allows you to know if the objective is completed

Reply:
yeah I think that I'm using that property, in this part I'm checking if all actions were completed if at least one is not completed I can't show the completed tag.

For what I saw this `isCompleted` is related to individual actions

Reply:
there is a `isCompleted` on `objectives` and on `actions` as well!

Reply:
@lucca65 ^^

Reply:
well, you see this particular task is about archiving **objectives**, not actions. the criteria to know if an objective is archived or not is the flag isCompleted.

the state of its child actions is not relevant to determine if an objective is completed or not, because not all actions can be completed, some of them will always have their status as false, because they can be recurrent, automatic or without uses

although it’s interesting to have this action counting and filtering this is not the criteria...

Reply:
to show the tag, just check the objetive.isCompleted property. actions don’t determine the status of the objective.